marimo/_cli/sandbox.py

```diff
@@ -34,6 +34,9 @@ def maybe_prompt_run_in_sandbox(name: str | None) -> bool:
     if name is None:
         return False
 
+    if Path(name).is_dir():
```

Comment:
I think we might want to log though to let the user know that sandbox won't work

Reply:
Is it the user expectation to enable sandbox if it's not explicitly set with a flag (`--sandbox`)?

Reply:
Was thinking along the lines of the explicit sandbox

```bash
$ uv run marimo edit --sandbox .
```

Suggesting `uv run` is good message I think.

but yes I think no log is fine for

```bash
$ uv run marimo edit .
```

Reply:
I made a longer issue describing some inconsistencies with sandboxing for multiple files: https://github.com/marimo-team/marimo/issues/5219